frontend/packages/kubevirt-plugin/integration-tests/tests/vm.detail.flavor.scenario.ts

```diff
@@ -27,10 +27,10 @@ describe('KubeVirt VM detail - edit flavor', () => {
   it(
     'changes tiny to large',
     async () => {
-      const vm1Config = vmConfig(configName.toLowerCase(), provisionConfig, testName);
+      const vm1Config = vmConfig(configName.toLowerCase(), testName, provisionConfig);
```

Comment:
Yes yes yes I agree 100%, I also hate that we need to pass the config twice, first to create the VirtualMachine object and then to the `create` method.
I'm not sure if this is the right time to do it though because we need to close one epic automation gap epic first and this would be a huge change.

Reply:
indeed, I agree it would be nicer but it is a big change in a late moment in the cycle. Lets address as a followup once the automation gap is closed.